src/useInterval/useInterval.ts

```diff
@@ -0,0 +1,36 @@
+import { onMounted, onUnmounted } from '@muban/muban';
+
+/**
+ *  A hook that can be used to call a function on a provided interval, by default the interval
+ *  will run immediate. You can also start and cancel the interval whenever needed.
+ *
+ * @param callback The callback you want to trigger once the interval runs.
+ * @param interval The duration of the interval you want to create.
+ * @param startImmediate Whether or not you want to immediately start the interval.
+ */
+export const useInterval = (
+  callback: () => void,
+  interval: number = 100,
+  startImmediate: boolean = true,
+): { startInterval: () => void; stopInterval: () => void } => {
+  let handle = -1;
```

Comment:
I'd give this variable the name `timeoutId`

Reply:
Wouldn't `intervalId` make more sense?